backend/danswer/background/celery/tasks/indexing/tasks.py

```diff
@@ -115,9 +115,11 @@ def check_for_indexing(self: Task, *, tenant_id: str | None) -> int | None:
                     server_host=INDEXING_MODEL_SERVER_HOST,
                     server_port=INDEXING_MODEL_SERVER_PORT,
                 )
-                warm_up_bi_encoder(
-                    embedding_model=embedding_model,
-                )
+                if old_search_settings:
```

Comment:
Nit: might be worth moving to the conditional above it? No need to construct the `EmbeddingModel` if we won't use it